scripts/setup_network_queues.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e
```

Comment:
What does this do?

Reply:
This script is intended to enlarge the size of Linux network queues. Default values are too low for our application.

Reply:
Removed. Will mention in documentation.